doc/source/whatsnew/v1.4.1.rst

```diff
@@ -25,7 +25,7 @@ Fixed regressions
 Bug fixes
 ~~~~~~~~~
 - Fixed segfault in :meth:``DataFrame.to_json`` when dumping tz-aware datetimes in Python 3.10 (:issue:`42130`)
--
+- Fixed failing join operations with overlapping ``IntervalIndex`` (:issue:`45661`)
```

Comment:
Is this a regression? If not please move to 1.5

Reply:
It works in 1.3.4. b49050736805a19fe95b39247f2cf037459971c1 broke it.

Reply:
Ok, plesae put it under regressions then

Reply:
Done